### DISCUSSION TITLE
Thanks for this gem!

### DISCUSSION THREAD

**Discussion by v-kolesnikov** [janko/sequel-activerecord_connection#23] [General]
Dear author!

I would like to express my warmest gratitude for your work!

**Comment by janko** [comment]
It makes me very happy to hear that ❤️ Out of curiosity, are you using it standalone, or as part of rodauth-rails?

**Reply by v-kolesnikov** [reply]
I even do not use it yet!
But I looked the code and found the great potential of this work. It makes me sure I could starting migrating my huge Rails codebase with lot of arel-based queries to Sequel. By the way I have already a huge experience with Sequel, Roda and Rodauth that all work perfect in contrast with Active-* family.
